src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx

```diff
@@ -277,28 +275,26 @@ describe("Base Configuration component", () => {
       baseConfigurationTypesSelect
     ).getByTitle("Open");
 
-    act(() => {
-      userEvent.click(baseConfigurationTypesButton);
-    });
+    userEvent.click(baseConfigurationTypesButton);
 
     expect(screen.getByText("Structure")).toBeInTheDocument();
     const target = screen.getByText("Structure");
 
-    act(() => {
-      userEvent.click(target);
-    });
+    userEvent.click(target);
+
+    // setting patient basis to true
```

Comment:
I think this should be false.

Reply:
Story specifies it to be true by default

Reply:
but it seems the following statement set it to false?

Reply:
ooh, this is the test case, my bad cool. I will update it.